contrib/babelfishpg_tds/src/backend/tds/tdsresponse.c

```diff
@@ -1251,6 +1264,60 @@ SendColumnMetadataToken(int natts, bool sendRowStat)
 			}
 		}
 
+		//For Spatial Types
+			tup = SearchSysCache1(TYPEOID, ObjectIdGetDatum(col->pgTypeOid));  //Search SysCache for type name
+			typTup = (Form_pg_type) GETSTRUCT(tup);
+
+			/*
+			 * Check if it is spatial Data type
+			 * Send the Corresponding MetaData Columns
+			*/			
+			if(strcmp(NameStr(typTup->typname),"geometry") == 0 || strcmp(NameStr(typTup->typname),"geography") == 0)
+			{
+				//Driver Expects assembly Name and Length 
+				//TODO: For Now assembly Name is Hardcoded but can be Altered in future
+				if(strcmp(NameStr(typTup->typname),"geometry") == 0){
+					assemblyName = "Microsoft.SqlServer.Types.SqlGeometry, Microsoft.SqlServer.Types, Version=11.0.0.0, Culture=neutral, PublicKeyToken=89845dcd8080cc91";
```

Comment:
nit: make this string a var or #define at top of file

Reply:
+1

Reply:
Done